src/components/navbar/__test__/Nav.test.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import Nav from '../Nav';
+import store from '../../../redux/store';
+
+test('It should render element with text COUNTRYPEDIA', async () => {
+  render(
+    <Provider store={store}>
+      <Nav />
+    </Provider>,
+  );
+  const headingElement = screen.getByText('COUNTRYPEDIA');
+  expect(headingElement).toBeInTheDocument();
+});
```

Comment:
- [ ] Some tests are failing. Please try to fix them 🙏 
![image](https://user-images.githubusercontent.com/46839548/217963143-dede877e-6a99-437f-87bb-75317757dc01.png)